components/movementsensor/rtk_station/rtk_station.go

```diff
@@ -179,46 +213,51 @@ func newRTKStation(
 
 	// Init movementsensor correction input addresses
 	r.logger.Debug("Init movementsensor")
-	r.serialPorts = make([]io.Writer, 0)
 
 	for _, movementsensorName := range r.movementsensorNames {
-		movementsensor, err := movementsensor.FromDependencies(deps, movementsensorName)
-		localmovementsensor := movementsensor
+		movementSensor, err := movementsensor.FromDependencies(deps, movementsensorName)
 		if err != nil {
 			return nil, err
 		}
+		rtkgps := movementSensor.(*gpsrtk.RTKMovementSensor)
```

Comment:
actually, do we need thhe rtkgps children anymore? since it's basically just writing to a serial port or an i2c address?

Reply:
Hm yeah..the only reason I could see it being needed is if some rovers are reading corrections from serial and some are reading from i2c...would that be possible?

Reply:
Okay to do it in a eparate pr.